addons/l10n_dk/models/account_chart_template.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    @api.model
+    def _prepare_transfer_account_for_direct_creation(self, name, company):
+        res = super(AccountChartTemplate, self)._prepare_transfer_account_for_direct_creation(name, company)
+        if company.country_id.code == 'DK':
+            account_tag_liquidity = self.env.ref('l10n_dk.account_tag_liquidity')
+            current_assets_type = self.env.ref('account.data_account_type_liquidity')
```

Comment:
@alyngsoe

Reply:
@jco-odoo  From an accounting point of view it can be discussed wether to set the internal transfer account as Liquidity og current assets.  We will accept both setups, i will let you to decide.

Reply:
@alyngsoe I removed the change of user type here (so it would remain current assets), but it is still added with the liquidity tag you defined.  Does that make sense?

Reply:
@jco-odoo Yes, that is perfect.